scripts/esbuild/util.ts

```diff
@@ -43,31 +46,16 @@ const externalNodeModules = [
   '@jest/core',
   '@jest/reporters',
   '@microsoft/typescript-etw',
-  'assert',
-  'buffer',
-  'child_process',
-  'console',
-  'constants',
   'expect',
   'fsevents',
-  'inspector',
   'jest',
   'jest-cli',
   'jest-config',
   'jest-message-id',
   'jest-pnp-resolver',
   'jest-runner',
-  'net',
   'puppeteer',
   'puppeteer-core',
-  'readline',
-  'stream',
-  'tty',
-  'url',
-  'util',
-  'vm',
-  'yargs',
-  'zlib',
```

Comment:
I neglected to read and heed my own comment on this variable! Basically if you run esbuild with `platform` set to `node` then it will automatically 1) externalize all node.js built-in modules and 2) it will automatically mark imports from those modules as side-effect free. This is important because it will allow dead-code-elimination to remove them. So! We actually don't want to have things like `crypto`, `buffer`, `util`, etc in here.